src/ReportCSVExporter.php

```diff
@@ -209,6 +209,7 @@ public function prepare_data_to_export() {
 		$rest_server = rest_get_server();
 		$report_data = $rest_server->response_to_data( $response, true );
 		remove_filter( 'woocommerce_rest_check_permissions', '__return_true' );
+		$report_data = $this->update_stock_status_for_export( $report_data );
```

Comment:
I'm not sure why this route was taken - in my previous comment I linked directly to the line that needed to be changed.

Reply:
Thanks @jeffstieler Updated https://github.com/woocommerce/woocommerce-admin/pull/7284/commits/6cdecbaf8c560ca0c2adf096e1beb34a7f8ff240

Reply:
It looks like I don't have to call the parent's constructor. Removed it in https://github.com/woocommerce/woocommerce-admin/pull/7284/commits/f3f56d89f857a08c76dd919788f38bfce57a6e46